tests/helpers/test_llm_stream.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import unittest
+from typing import AsyncIterator
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../..")))
+
+from iftk.helpers.llm import llm_stream
+
+MODEL_ID = "TinyLlama/TinyLlama-1.1B-Chat-v1.0"
+messages = [{"role": "user", "content": "How are you doing?"}]
+
+
+class TestLLM(unittest.IsolatedAsyncioTestCase):
+    async def test_llm(self):
+        async for token in llm_stream(model_id=MODEL_ID, messages=messages):
+            assert type(token) == str
```

Comment:
Prefer unittest asserts, in this case: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsInstance

```suggestion
            self.assertIsInstance(token, str)
```